.idea/codeStyles/Project.xml

```diff
@@ -72,15 +72,6 @@
       <option name="PACKAGES_TO_USE_STAR_IMPORTS">
         <value />
       </option>
-      <option name="PACKAGES_IMPORT_LAYOUT">
```

Comment:
This change may have had a diff in my environment as well, so I'll add it! Thank you!